src/reflect/scala/reflect/internal/pickling/UnPickler.scala

```diff
@@ -701,6 +705,7 @@ abstract class UnPickler {
       private val definedAtRunId = currentRunId
       private val p = phase
       protected def completeInternal(sym: Symbol) : Unit = try {
+        lazyCompletingSymbol = sym
```

Comment:
This assignment looks like it might be clobbered by a nested call to `completeInternal`. That nested call might not trigger a stub error at all, but subsequent code pertaining to this `sym` would still report the nested symbol.